packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -1,1286 +1,8 @@
-// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
copyright line should remain

(I'm a bit surprised that the `dev/bots/analyze.dart` check didn't complain here.)

Reply:
Right, thanks. I must've made a mistake when copy-pasting.